text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,183 @@
+# RFC: Feature Flag Advisor
+
+* Original Author(s): [Vivian Chen](https://quip-amazon.com/GGQ9EAAQ3Jw)
+* Tracking Issue: N/A
+* API Bar Raiser: [Rico Huijbers](https://quip-amazon.com/YPP9EA8Gt6Q)
+
+The feature flag CLI tool will inform users of their current feature flag configuration and allow them to switch any/all of them to a different value.
+
+## Working Backwards
+
+### **Help:** 
+
+```
+> cdk flags --help
+cdk flags 
+Finds and displays a report of the feature flag configuration and compares users' current values with our recommended values.
+ 
+Options:
+    --switch-all          **safely** switch all feature flags to their recommended states 
+    --switch-unconfigured **safely** switch unconfigured feature flags to their recommended states 
+    --toggle=string       toggle the state of a specific flag
+    
+Examples:
+    cdk flags
+    cdk flags --toggle="#FLAGNAME#"
+```
+
+### README:
+
+The new CLI command `cdk flags` shows you a report comparing your current feature flag configurations and our recommended states. When you run `cdk flags`, a feature flags report is displayed to you in the following format: flags that match the recommended values are at the top of the list, followed by flags that do not match the recommended values and then lastly, flags that are not configured by the user at all. There will be a Menu with options for the user to configure their feature flags. 
```

Comment:
Maybe leave out the "correct" flags altogether by default, hide them behind a `--all/-a` flag ?

Reply:
Also describe expected use cases here:

* You can see all your flags.
* If you want to do X, run the command C.
* Afterwards, if you want to do Y, run the command D.
* To get more information on a flag, you can do ...

Reply:
Then proceed to show the screens and explain a bit more.